enhancements/sig-policy/134-standalone-hub-templates/README.md

```diff
@@ -0,0 +1,185 @@
+# Standalone Hub Templates
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+To support Open Cluster Management (OCM) hub templates without the policy framework, this feature provides an additional
+`governance-standalone-hub-templating` addon which generates a user per cluster on the hub to be leveraged by the
```

Comment:
Does this mean cluster -> managedcluster on hub?

Reply:
Yes, the user is on the hub but unique per managed cluster.